TUTORIAL.md

```diff
@@ -127,7 +128,9 @@ Now, start it up...
 
 ### Run the Spring Boot App
 
-To make your Spring Boot app aware of your Okta settings, you need to set a few environment variables. You can also use [other options](https://docs.stormpath.com/java/servlet-plugin/config.html) for setting these variables.
+To make your Spring Boot app aware of your Okta settings, you need to set a few environment variables. You can also use [other options](https://docs.stormpath.com/java/servlet-plugin/config.html) for setting these variables. 
+
+// you said this already, look up there ^
```

Comment:
I was pointing it out so you could delete whichever you prefer. It doesn't seem like it fits better in either place, so your call.